src/debian/vmmgrapi/nginx.conf.j2

```diff
@@ -0,0 +1,44 @@
+worker_processes 1;
+user www-data www-data;
+error_log  /var/log/nginx/error.log warn;
+pid /var/run/nginx.pid;
+events {
+  worker_connections 1024;
+}
+http {
+  access_log /var/log/nginx/access.log combined;
+  upstream app_server {
+    server unix:/run/gunicorn.sock fail_timeout=0;
+  }
+  server {
+    listen {{ admin_cluster_ip }}:{{ vmmgr_http_port }} ssl;
+    ssl_certificate {{ vmmgrapi_certs_dir }}/seapath.crt;
+    ssl_certificate_key {{ vmmgrapi_certs_dir }}/seapath.key;
+
+    {% if vmmgr_http_api_acl is defined -%}
+    {{ vmmgr_http_api_acl | indent( width=4, first=False) }}
+    {%- endif %}
+
+    server_name seapath;
+    keepalive_timeout 5;
+    location / {
+      {% if vmmgr_http_local_auth_file is defined -%}
+      auth_basic           "RESTRICTED Area";
```

Comment:
Depending on the use case, there is also the auth_pam module that can be useful. With this module, you can use the system user and the LDAP user. It requires installing the package nginx-extras on Debian.